hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestSCMPipelineManager.java

```diff
@@ -253,10 +256,8 @@ public void testPipelineCreationFailedMetric() throws Exception {
       pipelineManager.createPipeline(HddsProtos.ReplicationType.RATIS,
           HddsProtos.ReplicationFactor.THREE);
       Assert.fail();
-    } catch (InsufficientDatanodesException idEx) {
-      Assert.assertEquals(
-          "Cannot create pipeline of factor 3 using 1 nodes.",
-          idEx.getMessage());
+    } catch (SCMException idEx) {
+      // pipeline creation failed this time.
```

Comment:
Do we have specific exception result code to check here?

Reply:
It doesn't have one before, while I can add some check here.